### ISSUE TITLE
Yuzu produces darker image than Ryujinx on Steam Deck in Gothic game

### ISSUE THREAD

**Issue by rafrafek** [yuzu-emu/yuzu#12844]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

1697

### Description of Issue

Image is darker than expected:

![image](https://github.com/yuzu-emu/yuzu/assets/23004737/f18c87a9-c24a-4be1-a1bd-a545c7ad3fa9)

### Expected Behavior

Ryujinx brightness / contrast / gamma - should be the same in Yuzu:

![image](https://github.com/yuzu-emu/yuzu/assets/23004737/9b4843b9-1b1d-4022-8af9-4a63abc9b426)

### Reproduction Steps

Install Yuzu and Ryujinx using EmuDeck on Steam Deck, launch Gothic 1 game in both emulators and compare the image.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/14085861/yuzu_log.txt)

### System Configuration

Steam Deck LCD
CPU: AMD Custom APU 0405 | AVX2 | FMA
GPU/Driver: AMD Custom APU 0405
RAM: 16.00 GiB
OS: Steam OS Preview channel

**Comment by Squall-Leonhart** [comment]
your steamdeck is adjusting the brightness based on either its own settings or a color management profile.

**Comment by rafrafek** [comment]
It happens only in Yuzu and only in Gothic. Other games in Yuzu look the same as in Ryujinx and their image is equally bright.

**Comment by liamwhite** [comment]
This issue is likely caused by the msaa resolve pass not dealing with srgb correctly

**Comment by Squall-Leonhart** [comment]
My bad, just got told the issue is due to the msaa pass not resolving correctly.

PS: don't use resolution scaling on this game, it uses formats not compatible with scaled copies.

**Comment by rafrafek** [comment]
Yeah, something is wrong with this game. It has only 24 fps in the menu, but somehow 60 fps in Ryujinx 🧐

Can I do something about the msaa and srgb?

**Comment by rafrafek** [comment]
Tested both emulators on the same Steam Deck but this time booting into Windows 11 and the results are exactly the same as on SteamOS.

**Comment by Squall-Leonhart** [comment]
its connected in part to the spammed "msaa uploads not supported" in the log

**Comment by rafrafek** [comment]
Is there a way to disable msaa? Or any other workaround?
